spec/storage/fog_spec.rb

```diff
@@ -36,5 +36,16 @@
         expect(subject.filename).to eq('foo.txt')
       end
     end
+
+    context "when url contains multi-byte characters ('日本語') in query string" do
+      before do
+        # Ruby 2.3.0 is returning a string with ASCII-8BIT encoding.
```

Comment:
This comment only confuses me. Why is the forced encoding required, and is this comment needed?